src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Corrupted_Native_x86_VS2013_Default.exe.sarif

```diff
@@ -50,6 +50,7 @@
           }
         }
       ],
+      "automationDetails": {},
```

Comment:
This is valid SARIF, automationDetails has no required members, *but* it's inefficient, we don't need to serialize an empty object. In general, we don't want our log file persistence to change version-over-version.

Reply:
fixed together in the SDK PR, link in the other comment with json node ordering.